plugins/woocommerce-admin/client/products/product-variation-form.tsx

```diff
@@ -44,44 +51,52 @@ export const ProductVariationForm: React.FC< {
 	}, [ productVariation ] );
 
 	return (
-		<Form< Partial< ProductVariation > >
-			initialValues={ productVariation }
-			errors={ {} }
-			ref={ formRef }
-		>
-			<ProductVariationFormHeader />
-			<ProductFormLayout key={ productVariation.id }>
-				<ProductFormTab name="general" title="General">
-					<ProductVariationDetailsSection />
-				</ProductFormTab>
-				<ProductFormTab name="pricing" title="Pricing">
-					<PricingSection />
-				</ProductFormTab>
-				<ProductFormTab name="inventory" title="Inventory">
-					<ProductInventorySection />
-				</ProductFormTab>
-				<ProductFormTab name="shipping" title="Shipping">
-					<ProductShippingSection
-						product={ productVariation as PartialProduct }
-					/>
-				</ProductFormTab>
-			</ProductFormLayout>
-			<ProductFormFooter />
+		<SlotContextProvider>
+			<Form< Partial< ProductVariation > >
+				initialValues={ productVariation }
+				errors={ {} }
+				ref={ formRef }
+			>
+				<ProductVariationFormHeader />
+				<ProductFormLayout key={ productVariation.id }>
+					<ProductFormTab name="general" title="General">
+						<ProductVariationDetailsSection />
+					</ProductFormTab>
+					<ProductFormTab name="pricing" title="Pricing">
+						<PricingSection />
+					</ProductFormTab>
+					<ProductFormTab name="inventory" title="Inventory">
+						<ProductInventorySection />
+					</ProductFormTab>
+					<ProductFormTab name="shipping" title="Shipping">
+						<WooProductSectionItem.Slot
+							location={ TAB_SHIPPING_ID }
```

Comment:
I am kind of pondering if we should be making this location variation tab specific. But we could leave it as is for now, and only change it if there is demand for having a field show up in shipping variations only?